tajo-docs/src/main/sphinx/tsql/variables.rst

```diff
@@ -28,35 +30,456 @@ Each client connection to TajoMaster creates a unique session, and the client an
+
+  * Property value: Boolean
+  * Default value: false
+  * Example
+
+.. code-block:: sh
+
+  \set PARTITION_NO_RESULT_OVERWRITE_ENABLED false
+
+.. describe:: TABLE_PARTITION_PER_SHUFFLE_SIZE
+
+In Tajo, storing a partition table is executed in two stages.
+This value indicates the output size of a task of the former stage, which determines the number of partitions to be shuffled between two stages.
+
+  * Property value: Integer
+  * Unit: MB
+  * Default value: 256
+  * Example
+
+.. code-block:: sh
+
+  \set TABLE_PARTITION_PER_SHUFFLE_SIZE 256
+
+.. describe:: ARITHABORT
+
+A flag to indicate how to handle the errors caused by invalid arithmetic operations. If true, a running query will be terminated with an overflow or a divide-by-zero.
+
+  * Property value: Boolean
+  * Default value: false
+  * Example
+
+.. code-block:: sh
+
+  \set ARITHABORT false
+
+.. describe:: MAX_OUTPUT_FILE_SIZE
+
+Maximum per-output file size. 0 means infinite.
+
+  * Property value: Integer
+  * Unit: MB
+  * Default value: 0
+  * Example
+
+.. code-block:: sh
+
+  \set MAX_OUTPUT_FILE_SIZE 0
+
+.. describe:: SESSION_EXPIRY_TIME
+
+Session expiry time.
+
+  * Property value: Integer
+  * Unit: seconds
+  * Default value: 3600
+  * Example
+
+.. code-block:: sh
+
+  \set SESSION_EXPIRY_TIME 3600
+
+.. describe:: CLI_COLUMNS
+
+Sets the width for the wrapped format.
+
+  * Property value: Integer
+  * Default value: 120
+  * Example
+
+.. code-block:: sh
+
+  \set CLI_COLUMNS 120
+
+.. describe:: CLI_NULL_CHAR
+
+Sets the string to be printed in place of a null value.
+
+  * Property value: String
+  * Default value: ''
+  * Example
+
+.. code-block:: sh
+
+  \set CLI_NULL_CHAR ''
+
+.. describe:: CLI_PAGE_ROWS
+
+Sets the number of rows for paging.
+
+  * Property value: Integer
+  * Default value: 100
+  * Example
+
+.. code-block:: sh
+
+  \set CLI_PAGE_ROWS 100
+
+.. describe:: CLI_PAGING_ENABLED
+
+Enable paging of result display.
+
+  * Property value: Boolean
+  * Default value: true
+  * Example
+
+.. code-block:: sh
+
+  \set CLI_PAGING_ENABLED true
+
+.. describe:: CLI_DISPLAY_ERROR_TRACE
+
+Enable display of error trace.
+
+  * Property value: Boolean
+  * Default value: true
+  * Example
+
+.. code-block:: sh
+
+  \set CLI_DISPLAY_ERROR_TRACE true
+
+.. describe:: CLI_FORMATTER_CLASS
+
+Sets the output format class to display results.
+
+  * Property value: Class name
+  * Default value: org.apache.tajo.cli.tsql.DefaultTajoCliOutputFormatter
+  * Example
+
+.. code-block:: sh
+
+  \set CLI_FORMATTER_CLASS org.apache.tajo.cli.tsql.DefaultTajoCliOutputFormatter
+
+.. describe:: ON_ERROR_STOP
+
+tsql will exist if an error occurs.
+
+  * Property value: Boolean
+  * Default value: false
+  * Example
+
+.. code-block:: sh
+
+  \set ON_ERROR_STOP false
+
+.. describe:: NULL_CHAR
+
+Null char of text file output.
```

Comment:
It would be better to add 'used when the table property ``text.null`` is not specified' after the this description.